### ISSUE TITLE
MAKING EVERY BIODOME IN TO AN ISLAND.

### ISSUE THREAD

**Issue by Pcskiller** [TheGameCreators/GameGuruRepo#4123]
Hi, i wanted to ask this for a long time, i haven't spend a lot of time lately with max so if this  feature is available then i apologize for this.
Can you please and an extra feature that after the player has selected his biodome and the size , that just before the word are created that there is an option available that you can  tick if you want that biodome to be an island weather it is a dessert of cliff or snow or jungle biodome , it then gets created as an island meaning that instead of having endless terrain as far as the eye can see that you just generate only the ocean  all around the world  , so every biodome can be a island or just a normal endless mountain terrain like now.

i think it would not be so difficult one just take the endless terrain away then you automatically have and island word with just see around it.

**Comment by Pcskiller** [comment]
can we have some  feedback on this. 
this would be an easy to add feature.
this will be an game ganging idea
please can we have this as soon as possible.

**Comment by Pcskiller** [comment]
any feedback please we need this.

**Comment by LeeBamberTGC** [comment]
@Pcskiller Yes we just ran out of time running up to V1 to add this small addition.  The only way right now would be to use Preben's Easy Terrain Tool and import a height map in the shape of an island, but I agree the ideal solution is for the biome to be modified to drop the edges into the sea with some additional settings.  For any C++ coders out there, we are currently on a bug fixing run, but if you wish to check out the code and make this a thing, you will the repo here: https://github.com/TheGameCreators/GameGuruMAX

**Comment by Pcskiller** [comment]
im still waiting for this ? i don't think it would be ideal for the people to import a height map, if this can just be added, so I'm not giving up on this I'm waiting please make this happened.

**Comment by Pcskiller** [comment]
so bug fixing has gone well, bugs are down a lot, how about giving us this function ? please man.
